src/lib/queryVersions.ts

```diff
@@ -70,7 +70,7 @@ async function queryVersions(packageMap: Index<VersionSpec>, options: Options =
         ...options,
         // upgrade prereleases to newer prereleases by default
         distTag,
-        pre: options.pre != null ? options.pre : isPre(version),
+        pre: options.pre != null ? options.pre : distTag !== 'latest' || isPre(version),
```

Comment:
Considered adding the check into isPre, as the second argument, but seeing how isPre is used that was a bad idea.
In all the other usage locations, the checks are made against version to determine if it is has a prefix. And there is no distTag in that context